### ISSUE TITLE
h2 styling

### ISSUE THREAD

**Issue by rimmesbe** [ekitnat/Codeacademy#5]
only your first h2 is being styled correctly. All this styling except the background image should be shared by all the h2's. Try and figure out how to use 1 selector to give this styling to all the h2's.

https://github.com/ekitnat/Codeacademy/blob/master/Dasmotos/resources/css/style.css#L13-L19

**Comment by ekitnat** [comment]
do you mean that this background-color: mediumspringgreen; code should be shared in all h2 tags?

**Comment by rimmesbe** [comment]
no, the color white, font-size etc. The shared stylings. The background color is the only unique style for each.
